library/ui-strings/src/main/res/values/strings.xml

```diff
@@ -3417,6 +3417,10 @@
     <string name="labs_enable_client_info_recording_summary">Record the client name, version, and url to recognise sessions more easily in session manager.</string>
     <string name="labs_enable_voice_broadcast_title">Enable voice broadcast (under active development)</string>
     <string name="labs_enable_voice_broadcast_summary">Be able to record and send voice broadcast in room timeline.</string>
+    <string name="labs_enable_tablet_mode_title">Enable tablet mode (split screen)</string>
+    <string name="labs_enable_tablet_mode_summary">Allow Chats to be opened in split window.</string>
+    <string name="labs_enable_multichat_title">Enable multiwindow chats</string>
+    <string name="labs_enable_multichat_summary">Allow Multiple Chats to be opened in split windows (need tablet mode enabled).</string>
```

Comment:
Please use strings_sc.xml when contributing to SchildiChat, the upstream strings.xml will be cleaned on each upstream merge.